eng/config/PublishData.json

```diff
@@ -78,7 +78,7 @@
       "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
       "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
       "channels": [ "dev16.7", "dev16.7p3" ],
-      "vsBranch": "master",
```

Comment:
PublishData.json is always pulled from master, so it needs to be updated here (so it also has the correct branch name)
https://github.com/dotnet/roslyn/blob/master/eng/config/PublishData.json#L140

I think this got messed up in one of the merges I did... my bad!